src/irmin/node_intf.ml

```diff
@@ -73,12 +73,23 @@ module type S = sig
   (** [default] is the default metadata value. *)
 end
 
+module type Metadata = sig
```

Comment:
Why is this module type duplication necessary?

Reply:
Indeed that was a mistake (not sure how this change was included at the first place!)